### ISSUE TITLE
Notification sound

### ISSUE THREAD

**Issue by MGRashed** [basildane/WakeOnLAN#156]
I have set the "Enable Sound" = False, but still the pinger sound is occurring whenever a PC comes to online or goes offline.

**Comment by basildane** [comment]
What version are you running?  I just tested it and it is working here.

**Comment by MGRashed** [comment]
Latest version. Is it related to Windows 10 notification or WakeOnLan's own notification?

**Comment by basildane** [comment]
It is local to WOL, it just plays a sound.  I've never experienced this problem.

**Comment by MGRashed** [comment]
I have just uninstalled, and then reinstalled it. Still the same, notifications are happening even though I have set it to be False in the Options menu.

**Comment by basildane** [comment]
Could you send me some diagnostic info?  Open the WOL program, click on Tools -> Trace Log.  This will create a text file called My Documents/wakeonlan.tracelog.

Please upload that file to me at [Uploads](https://cloud.aquilatech.com/s/cKHVPHPXfaYdbjn)

At this point I have no idea why this would be happening.

**Comment by MGRashed** [comment]
Just uploaded the file you requested, let's hope this issue will be resolved shortly.

**Comment by basildane** [comment]
I reviewed your settings, which appear correct.  And I ran a test with some servers here, and I'm not able to duplicate the problem.  One thing I did notice is that your host keeps appearing to go online/offline very frequently.

Unfortunately I have no idea why this is happening.
Have you tried uninstalling and reinstalling WOL?

**Comment by MGRashed** [comment]
> Have you tried uninstalling and reinstalling WOL?

Yes I did it yesterday, same situation. I have an unstable network (I think router issue), so the host is dropping out often. But that should not trigger WOL notification when I have set it to False in options, right?

**Comment by basildane** [comment]
no, it should not beep if sound is set to false.  I tested it, and I even reviewed the source code.  I'm stumped.  It should not be possible.

**Comment by MGRashed** [comment]
What I noticed when reinstalling yesterday is I did not need to import the settings afterwords, it just took it automatically from somewhere. Where is the settings usually stored? Maybe I have to do a clean reinstallation by deleting everything in the system related to WOL?

**Comment by basildane** [comment]
First, go to options and look at "database path".
That will tell you where your machines.xml file is located, which lists all your hosts and their parameters.

Next, look in the folder C:\ProgramData\Aquila Technology\WakeOnLAN\
There will be subfolders for each version of WOL you have run.  You may exit the program and delete this whole folder tree.

Then restart the application.

**Comment by MGRashed** [comment]
I have uninstalled it, and deleted both folders in ProgramData and Program Files, and finally installed it followed by the Settings being imported. I will comment back in few days to see how it went.

One thing to note, the uninstaller did not remove anything from the Program Files, perhaps that was the issue (left over files causing problem)?

**Comment by basildane** [comment]
Uninstall should leave the ProgramData folder untouched.  This is so your settings will automatically carry over when you upgrade or reinstall.
